datafusion/common/src/config.rs

```diff
@@ -454,6 +465,87 @@ config_namespace! {
     }
 }
 
+#[cfg(feature = "parquet")]
```

Comment:
Can you please  move this into some other module (so we don't have parquet stuff end up in normal config?)